src/SonarLint.VSSpecificAssemblies.props

```diff
@@ -111,4 +111,13 @@
     </Reference>
   </ItemGroup>
 
+
+  <!-- ******************************************************************************* -->
+  <!-- MS.VS.LanguageServices package -->
+  <!-- ******************************************************************************* -->
+  <ItemGroup Label="MS.VS.LanguageServices" Condition="$(RequiresLanguageServices)=='true'">
+    <!-- This package references Roslyn components used by VS. It isn't part of the VSSDK meta-package. -->
+    <PackageReference Include="Microsoft.VisualStudio.LanguageServices" Version="1.3.2" />
+  </ItemGroup>
```

Comment:
Strictly speaking this import file is called `VSSpecificAssemblies`, but this package reference isn't version-specific i.e. we can use the same version in VS2019 and VS2022. However, we don't currently have a better location for this kind of shared reference, and the ALM reference above is not version-specific either.